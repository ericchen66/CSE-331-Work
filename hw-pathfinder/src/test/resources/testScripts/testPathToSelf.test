# Tests Dijkstra's algorithm for a node
# with a path to itself

CreateGraph simpleGraph

AddNode simpleGraph A

FindPath simpleGraph A A