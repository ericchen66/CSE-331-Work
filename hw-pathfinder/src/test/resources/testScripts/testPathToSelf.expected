# Tests Dijkstra's algorithm for a node
# with a path to itself

created graph simpleGraph

added node A to simpleGraph

path from A to A:
total cost: 0.000